packages/flutter/test/material/dropdown_test.dart

```diff
@@ -63,18 +63,25 @@ class TestApp extends StatefulWidget {
 class _TestAppState extends State<TestApp> {
   @override
   Widget build(BuildContext context) {
-    return new MediaQuery(
-      data: new MediaQueryData.fromWindow(window),
-      child: new Directionality(
-        textDirection: widget.textDirection,
-        child: new Navigator(
-          onGenerateRoute: (RouteSettings settings) {
-            assert(settings.name == '/');
-            return new MaterialPageRoute<dynamic>(
-              settings: settings,
-              builder: (BuildContext context) => widget.child,
-            );
-          },
+    return new Localizations(
+      locale: const Locale('en', 'US'),
+      delegates: <LocalizationsDelegate<dynamic>>[
+        DefaultWidgetsLocalizations.delegate,
+        DefaultMaterialLocalizations.delegate,
+      ],
+      child: new MediaQuery(
+        data: new MediaQueryData.fromWindow(window),
+        child: new Directionality(
```

Comment:
ditto

Reply:
This one is awkward to remove. The test verifies that things work with different textDirections. While Localizations does provide a text direction, I cannot modify it directly. I could rewrite the test to work on different locales instead, but I think that would hide the real purpose of the test (which is to make sure that different text directions work). Therefore, I left the Directionality widget in the tree.

Reply:
that's fine. i didn't notice it was being tweaked.